Database/Scripts/Builders/datapoints-builder.py

```diff
@@ -0,0 +1,97 @@
+import json
+
+with open('./Scraped JSONs/datasets.json') as f:
```

Comment:
We can remove them in that case but for the purpose of helping others, keeping them would mean they can access the JSON files without needing to run the scripts. It can be either way, of course my preference is to keep them to simplify it for others and to have people see the files that the scripts created. Also, if we need to run the scripts again incase of a bug or other, it would require less work on our part to run everything again

Reply:
We dont need to keep these since we are only running them once to populate the DB. Any technically inclined user will be able to run the scripts in sequential order here

Reply:
I would keep at least **datasets-with-datapoints.json** as it's been used many times to check against errors in the database or to verify that all the data has been parsed / inserted correctly. 
We also use websites like [JSON Viewer](http://jsonviewer.stack.hu/) to view this JSON is an organized way.
Since this JSON contains the 387 datasets with their corresponding data points, it might be used in the future for debugging purposes